src/sentry/static/sentry/app/views/integrationPipeline/awsLambdaFunctionSelect.tsx

```diff
@@ -148,7 +148,8 @@ const Wrapper = styled('div')`
   padding: 100px 50px 50px 50px;
 `;
 
-const StyledForm = styled(Form)`
+// TODO(ts): Understand why styled is not correctly inheriting props here
+const StyledForm = styled(Form)<Form['props']>`
```

Comment:
Not sure what's going on here, but without being explicit tsc thinks the StyledForm doesn't have any props 😬